### PR TITLE
added searchbar

### LINKED ISSUE #31: Centre the Navigation links and add search bar
The navigation links should be in the centre and a search bar added. See screenshot
image

### PR DESCRIPTION
**Pull request by padmajabhol**

### CODE REVIEW

**Review by JPTIZ — APPROVED** [review]
It has a small visual issue on Firefox as the browser highlights the selected input, and since the input and the search icon are separated it does not highlight the icon part.

https://user-images.githubusercontent.com/11988816/136670911-113c3006-de9b-463e-a71f-a348fc121be9.mp4

Maybe it is slightly complicated to fix, but I think this PR itself does cover what the issue requested, so I'll let it approved and wait for another approval :) Thank you for your first contribution on this repo!

**Review by ikayz — APPROVED** [review]

**Pull request merged**
